theano/tensor/nnet/nnet.py

```diff
@@ -29,6 +29,7 @@
 from theano.tensor.nnet.sigm import sigmoid, softplus
 from theano.gradient import DisconnectedType
 from theano.gradient import grad_not_implemented
+from theano.sandbox.blocksparse import sparse_block_dot
```

Comment:
I agree, let's move blocksparse.py out of sandbox. But obviously sandbox/cuda/blocksparse.py will stay there.